packages/node_modules/@cerebral/inferno/README.md

```diff
@@ -0,0 +1,55 @@
+# @cerebral/inferno
+
+[Inferno](http://infernojs.org) view for cerebral.
+
+**NPM**
+
+`npm install @cerebral/inferno inferno inferno-component inferno-create-element babel-plugin-inferno`
+
+```js
+import Inferno from 'react'
```

Comment:
Is this correct?

Reply:
I just copied it from docs/api/components.md, but you're right it doesn't look correct.